sections/rich-text.liquid

```diff
@@ -9,7 +9,7 @@
     {%- for block in section.blocks -%}
       {%- case block.type -%}
         {%- when 'heading' -%}
-          <h2 class="{% if block.settings.heading_size == 'small' %}h2{% else %}h1{% endif %}" {{ block.shopify_attributes }}>{{ block.settings.heading | escape }}</h2>
+          <h2 class="{% if block.settings.heading_size == 'large' %}h1{% elsif block.settings.heading_size == 'medium' %}h2{% else %}h3{% endif %}" {{ block.shopify_attributes }}>{{ block.settings.heading | escape }}</h2>
```

Comment:
I don't know why but the option doesn't output the right heading size, they look too small now. After investigating it seems like the options before were already perfect and using respectively H1 and H2, I am sorry for this confusion! Large should be using the H0. [It was planned to offer a taller heading size](https://screenshot.click/21-07-acjh4-kqqie.png) but I don't recall what happened to this exploration. It would be key for the theme to have this option. 

Keep the size like before for `Small` and `Medium` but please add a new heading style H0 for large heading to apply to the `Large` option:

- Desktop is 52px
- Mobile is 40px


cc.  @nicklepine @Oliviammarcello @wiktoriaswiecicka, I couldn't find the updated version of the new Typography changes like it was supposed to be done to reference in those circumstances. We'll need this to inform and support our reviews please.